_data/install.yaml

```diff
@@ -70,6 +70,10 @@
   url: /install/on_freebsd
   caption: FreeBSD
   icon: install/freebsd@2x.png
+-
+  url: /install/on_openbsd
+  caption: OpenBSD
+  icon: install/openbsd@2x.png
```

Comment:
I don't think this image is available.
If it is added, I strongly prefer using an SVG (https://github.com/crystal-lang/crystal-website/pull/91#issuecomment-545033445)
/cc @bcardiff

Reply:
Using svg is a separate discussion. We can try to get svg icons for icons later